packages/grid_client/scripts/applications/casberlabs.ts

```diff
@@ -0,0 +1,142 @@
+import { FilterOptions, GatewayNameModel, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log } from "../utils";
+
+async function pingNodes(client, nodes) {
+  for (const node of nodes) {
+    try {
+      await client.zos.pingNode({ nodeId: node.nodeId });
+      return node.nodeId;
+    } catch (error) {
+      console.log("node " + node.nodeId + " is not responding, trying different node.");
```

Comment:
throw error instead of logging

Reply:
Throwing the error will stop the script from running on the first unresponsive node (which happens a lot), and it's kind of neglecting the try and catch.